### DISCUSSION TITLE
Send Button

### DISCUSSION THREAD

**Discussion by Metairieman55** [LykosAI/StabilityMatrix#1007] [Q&A]
Just as a curiosity . . . I'm not a code-type person. What is the "Send" box at the bottom of the "Packages" screen? Hadn't seen any reference to it anywhere. I've been successfully using the program for a few months  now, too.

Thanks!

**Comment by mohnjiles** [comment]
Hmm, I don't recall a 'Send' button in that spot... could you share a screenshot so we can take a look? Thanks!
